### ISSUE TITLE
Could not solve for environment specs for dragonflye-1.0.13

### ISSUE THREAD

**Issue by lskatz** [bactopia/bactopia#401]
**Description**
After running `bactopia download --build-all envtype conda`, I get an error that the environment cannot be solved.

**Steps to Reproduce**
```
(bactopia-dev) [gzu2@monolith3 tmp]$ bactopia download --build-all envtype conda
2023-03-22 16:20:58 INFO     2023-03-22 16:20:58:root:INFO - Checking if environment pre-builds are needed (this may take a    download.py:538
                             while if building for the first time)
                    INFO     2023-03-22 16:20:58:root:INFO - Begin gather create to                                            download.py:255
                             /scicomp/home-pure/gzu2/.bactopia/conda/bioconda--bactopia-gather-1.0.1
2023-03-22 16:23:44 INFO     2023-03-22 16:23:44:root:INFO - Begin qc create to                                                download.py:255
                             /scicomp/home-pure/gzu2/.bactopia/conda/bioconda--bactopia-qc-1.0.0
2023-03-22 16:31:50 INFO     2023-03-22 16:31:50:root:INFO - Begin assembler create to                                         download.py:255
                             /scicomp/home-pure/gzu2/.bactopia/conda/bioconda--bactopia-assembler-1.0.1
2023-03-22 16:32:07 ERROR    2023-03-22 16:32:07:root:ERROR - Error creating Conda environment, retrying after short sleep.    download.py:368
2023-03-22 16:32:54 ERROR    2023-03-22 16:32:54:root:ERROR - Error creating Conda environment, retrying after short sleep.    download.py:368
2023-03-22 16:34:28 ERROR    2023-03-22 16:34:28:root:ERROR - Error creating Conda environment, retrying after short sleep.    download.py:368
2023-03-22 16:36:15 ERROR    2023-03-22 16:36:15:root:ERROR - Error creating Conda environment, retrying after short sleep.    download.py:368
2023-03-22 16:38:32 ERROR    2023-03-22 16:38:32:root:ERROR - Error creating Conda environment, retrying after short sleep.    download.py:368
2023-03-22 16:41:19 ERROR    2023-03-22 16:41:19:root:ERROR - External command failed with exit code 1!                            utils.py:43

                             Command:
                             bash -c 'mamba create -y -p
                             /scicomp/home-pure/gzu2/.bactopia/conda/bioconda--bactopia-assembler-1.0.1 -c conda-forge -c bioconda
                             --force bioconda::bactopia-assembler=1.0.1'

                             Standard output:
                             conda-forge/linux-64                                        Using cache
                             conda-forge/noarch                                          Using cache
                             bioconda/linux-64                                           Using cache
                             bioconda/noarch                                             Using cache
                             bioconda/linux-64                                           Using cache
                             bioconda/noarch                                             Using cache

                             Looking for: ['bioconda::bactopia-assembler=1.0.1']

                             Could not solve for environment specs
                             Encountered problems while solving:
                               - package dragonflye-1.0.13-hdfd78af_0 requires medaka >=1.5.0, but none of the providers can be
                             installed

                             The environment can't be solved, aborting the operation

```

**Expected Behavior**
A solved conda environment

**Execution Environment**

- Bactopia Version: bactopia 3.0.0
- OS: $ lsb_release -a  
LSB Version:    :core-4.1-amd64:core-4.1-noarch  
Distributor ID: CentOS  
Description:    CentOS Linux release 7.9.2009 (Core)  
Release:        7.9.2009  
Codename:       Core  
- Environment: conda

**Additional Information**
I had removed all previous conda and singularity environments just prior to this to be sure it would not be affected by anything previous.

```
mv /scicomp/home-pure/gzu2/.bactopia/conda /scicomp/home-pure/gzu2/.bactopia/conda.202-03-22
mv /scicomp/home-pure/gzu2/.bactopia/singularity /scicomp/home-pure/gzu2/.bactopia/singularity.2023-03-22
```

**Comment by lskatz** [comment]
Changing the channel priority policy from `strict` to `flexible` seemed to fix it.

**Comment by rpetit3** [comment]
Thanks @lskatz on the Dragonflye end, I released a v1.1.0. The hope was to split dragonflye into `dragonflye` and `dragonflye-gpu`. But there is currently an issue when solving envs that use `tensorflow-gpu` and missing `__cuda` libraries.

So, going forward, I've dropped direct GPU support on Dragonflye. The advanced users that use GPUs can easily add `tensorflow-gpu` to the `conda create` command.

I'm hoping this will fix the conflicts we are having.
